cirq/ops/common_channels.py

```diff
@@ -36,8 +36,8 @@ def __init__(self,
                  p_x: Optional[float] = None,
                  p_y: Optional[float] = None,
                  p_z: Optional[float] = None,
-                 error_probabilities: Optional[Dict[str, float]] = None
-                ) -> None:
+                 error_probabilities: Optional[Dict[str, float]] = None,
+                 tol: Optional[float] = None) -> None:
```

Comment:
BTW: Maybe let's use 1e-8 as the default for consistency with other code in cirq.